e2e-test/fluxd.yaml

```diff
@@ -0,0 +1,166 @@
+metadata:
+  labels:
+    name: flux
+  name: flux
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: flux
+subjects:
+  - kind: ServiceAccount
+    name: flux
+    namespace: default
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flux-recv
+data:
+  fluxrecv.yaml: |
+    fluxRecvVersion: 1
+    endpoints: []
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flux
+  namespace: default
+  labels:
+    name: flux
+    type: deployment
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
+  selector:
+    matchLabels:
+      name: flux
+  template:
+    metadata:
+      labels:
+        name: flux
+        type: deployment
+    spec:
+      serviceAccountName: flux
+      volumes:
+        - name: git-key
+          secret:
+            secretName: flux-git-deploy
+            defaultMode: 0400 # when mounted read-only, we won't be able to chmodª
+        - name: fluxrecv-config
+          configMap:
+            name: flux-recv
+            defaultMode: 0400
+        - name: git-keygen
+          emptyDir:
+            medium: Memory
+        - name: tmp
+          emptyDir:
+            medium: Memory
+        - name: kubedir
+          configMap:
+            name: flux-kube-config
+        - name: sourcegitrepo
+          hostPath:
+            path: /sourcegitrepo
+      containers:
+        - name: flux
+          image: docker.io/fluxcd/flux:1.18.0
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              cpu: 50m
+              memory: 64Mi
+          ports:
+            - containerPort: 3030 # informational
+          env:
+            - name: KUBECONFIG
+              value: /root/.kubectl/config
+          volumeMounts:
+            # - name: git-key
+            #   mountPath: /etc/fluxd/ssh
+            #   readOnly: true
+            - name: git-keygen
+              mountPath: /var/fluxd/keygen
+            - name: tmp
+              mountPath: /tmp
+            - name: kubedir
+              mountPath: /root/.kubectl
+            - name: sourcegitrepo
+              mountPath: /sourcegitrepo
+          args:
+            - --ssh-keygen-dir=/var/fluxd/keygen
+            - --git-readonly=true
+            - --git-url=file:///sourcegitrepo
+            - --git-branch=master
+            - --git-path=local/releases
+            - --git-poll-interval=5s
+            - --sync-interval=5s
+            - --listen-metrics=:3031
+            - --sync-garbage-collection=true
+            - --registry-disable-scanning
+        - name: recv
+          image: fluxcd/flux-recv:0.3.0
+          imagePullPolicy: IfNotPresent
+          args:
+            - "--config=/etc/fluxrecv/fluxrecv.yaml"
+          ports:
+            - containerPort: 8080
+              hostPort: 8080
+          volumeMounts:
+            - name: fluxrecv-config
+              mountPath: /etc/fluxrecv
```

Comment:
`recv` won't work with hooks when running here, will it? In that case we might as well remove it entirely?

Reply:
Yeah, we could consider removing it. I was considering adding a fake webhook solution a some point

Reply:
With a sync interval of 5s it might not be that relevant. :) except if the events on the websocket are different on this trigger than of normal changes.

Reply:
Yeah, last part was also my thought. But lets try to remove it.
What about the release-manager, does it support pulling too? Or is it only webhookable?